control/responsible_role.go

```diff
@@ -0,0 +1,84 @@
+package control
+
+import (
+	"github.com/jbowtie/gokogiri/xml"
+	"errors"
+	"regexp"
+	"strings"
+	"fmt"
+)
+
+// FindResponsibleRole looks for the Responsible Role cell in the control table.
+func FindResponsibleRole(ct *Table) (*ResponsibleRole, error) {
+	nodes, err := ct.searchSubtree(".//w:tc//w:t[contains(., 'Responsible Role')]")
+	if err != nil {
+		return nil, err
+	}
+	if len(nodes) != 1 {
+		return nil, errors.New("could not find Responsible Role cell")
+	}
+	// Not sure why we have to get the parent's parent, but we need to.
+	// If we only go up once, it won't find the other text nodes.
+	parentNode := nodes[0].Parent().Parent()
```

Comment:
Hmm, just realizing that this wouldn't cover the case where `Responsible` and `Role` are split between `<w:t>` tags. Maybe it can be simplified to something like

```go
parentNode, err := ct.searchSubtree(".//w:tc[starts-with(normalize-space(.), 'Responsible Role')]")
```